package-lock.json

```diff
@@ -0,0 +1,17491 @@
+{
+  "requires": true,
```

Comment:
Delete this file.

Use `yarn` to install any new dependencies:

```
yarn add react-native-paper react-native-dropdown-picker
```

Reply:
should be resolved